datasources.go

```diff
@@ -36,6 +42,16 @@ func (api API) CreateDataSource(name string) (*DataSource, error) {
 	return ds, err
 }
 
+// CreateDataSource2 creates an API Data Source in ChartMogul.
+// * Allows other parameters than just the name.
+//
+// See https://dev.chartmogul.com/v1.0/reference#data-sources
+func (api API) CreateDataSource2(dataSource *DataSource) (*DataSource, error) {
```

Comment:
`CreateDataSource2` is a terrible name. Can you think of something better than communicates the intent of the function? Otherwise anyone using this is going to have to read the docs or the source to work out the difference between `CreateDataSource` and `CreateDataSource2`